### DISCUSSION TITLE
Is it possible to have file level attributes?

### DISCUSSION THREAD

**Discussion by zetashift** [jgm/djot#315] [Q&A]
Is there a way to put metadata on the first lines of a file to indicate 'file properties', kind of like YAML frontmatter for markdown or the following org snippet:
```
#+title:  A cool title?
```

Couldn't find anything in the syntax description or in the issues(well I searched for 'file attributes')

**Comment by yurivish** [comment]
Related: https://github.com/jgm/djot/issues/35 (metadata can be seen as an [alternative](https://github.com/jgm/djot/issues/35#issuecomment-1661630557) to Markdown-style frontmatter)

**Comment by irskep** [comment]
I used Markdown-style frontmatter in Djockey: https://steveasleep.com/djockey/basics/frontmatter.html

Feels like a hack since it just chops off the front matter and parses it with YAML before sending to the Djot parser, but it works.
